### DISCUSSION TITLE
Crew command not working after attempting to install chrome brew.

### DISCUSSION THREAD

**Discussion by samsibila** [skycocker/chromebrew#9757] [Q&A] [answered]
I've been trying to install chromebrew on my Chromebook for a while and trying everyone's methods online but no luck, I'm new to chromos but not to computers and this is the first of any sort of "public plea of help" I've made via the forum so I'm unsure what all I need to provide here to make sense of my problem

sudo chromeos-setdevpasswd
sudo crossystem dev_boot_signed_only=1
I run this as it said to create a passwd for chromos

bash <(curl -L git.io/vddgY) && . ~/.bashrc
now I ran that after I loggen in as chomos and it said if I were to do this it would wipe out all files in /usr/local and gives me a yes/no option, when I type in yes the prompt goes away and it looks like it worked

but when I then try to run a crew command it says the command is unavailable.

can anyone help me out with this?

**Comment by samsibila** [comment]
ok update to this, I am actually unable to install chromebrew becaule the /usr/local directory had things added from the dev_install script I ran. is there any way someone was able to figure out if this is a bug or not because for me it will not let me install it.

**Reply by uberhacker** [reply, marked as answer]
Is there a reason to save anything from `dev_install`?  If not, I would wipe everything and start with a clean slate.
```
$ rm -rf /usr/local
$ bash <(curl -L git.io/vddgY) && . ~/.bashrc
```

**Reply by samsibila** [reply]
thank you! this did work for me. I can also say now that I have installed chromebrew is there any like videos you would recomend to watch to get familiar with what chromebrew has to offer? my personal goal at some point is to find a way to use the "macchanger" command on my Chromebook but I am unsure what is available to do so. if you know anyone who could help with this it would be great.

**Reply by uberhacker** [reply]
Not that I'm aware but if you try to get familiar with a few commands, that will help you along.  Here is a short list to get you started.

| Command      | Description |
|:------------:|:------------|
| help         | get information about command usage |
| install      | install package(s) along with dependencies after prompting for confirmation |
| remove       | remove package(s) |
| search       | look for package(s) |
| update       | update crew itself |
| upgrade      | update all or specific package(s) |

These are the commands you will likely use 90% of the time.
